### ISSUE TITLE
Add a little obfuscation by encoding the payload?

### ISSUE THREAD

**Issue by santosomar** [RealJTG/Meltdown#7]
Perhaps add a little spice to it ;-) LOL

`ICAgIFdlJ3JlIG5vIHN0cmFuZ2VycyB0byBsb3ZlLA0KICAgIFlvdSBrbm93IHRoZSBydWxlcyBhbmQgc28gZG8gSSwNCiAgICBBIGZ1bGwgY29tbWl0bWVudCdzIHdoYXQgSSdtIHRoaW5raW5nIG9mLA0KICAgIFlvdSB3b3VsZG4ndCBnZXQgdGhpcyBmcm9tIGFueSBvdGhlciBndXksDQogICAgLA0KICAgIEkganVzdCB3YW5uYSB0ZWxsIHlvdSBob3cgSSdtIGZlZWxpbmcsDQogICAgR290dGEgbWFrZSB5b3UgdW5kZXJzdGFuZCwNCiAgICAsDQogICAgTmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXAsDQogICAgTmV2ZXIgZ29ubmEgbGV0IHlvdSBkb3duLA0KICAgIE5ldmVyIGdvbm5hIHJ1biBhcm91bmQgYW5kIGRlc2VydCB5b3UsDQogICAgTmV2ZXIgZ29ubmEgbWFrZSB5b3UgY3J5LA0KICAgIE5ldmVyIGdvbm5hIHNheSBnb29kYnllLA0KICAgIE5ldmVyIGdvbm5hIHRlbGwgYSBsaWUgYW5kIGh1cnQgeW91LA0KICAgICwNCiAgICBXZSd2ZSBrbm93biBlYWNoIG90aGVyIGZvciBzbyBsb25nLA0KICAgIFlvdXIgaGVhcnQncyBiZWVuIGFjaGluZyBidXQgeW91J3JlIHRvbyBzaHkgdG8gc2F5IGl0LA0KICAgIEluc2lkZSB3ZSBib3RoIGtub3cgd2hhdCdzIGJlZW4gZ29pbmcgb24sDQogICAgV2Uga25vdyB0aGUgZ2FtZSBhbmQgd2UncmUgZ29ubmEgcGxheSBpdCwNCiAgICAsDQogICAgQW5kIGlmIHlvdSBhc2sgbWUgaG93IEknbSBmZWVsaW5nLA0KICAgIERvbid0IHRlbGwgbWUgeW91J3JlIHRvbyBibGluZCB0byBzZWUsDQogICAgLA0KICAgIE5ldmVyIGdvbm5hIGdpdmUgeW91IHVwLA0KICAgIE5ldmVyIGdvbm5hIGxldCB5b3UgZG93biwNCiAgICBOZXZlciBnb25uYSBydW4gYXJvdW5kIGFuZCBkZXNlcnQgeW91LA0KICAgIE5ldmVyIGdvbm5hIG1ha2UgeW91IGNyeSwNCiAgICBOZXZlciBnb25uYSBzYXkgZ29vZGJ5ZSwNCiAgICBOZXZlciBnb25uYSB0ZWxsIGEgbGllIGFuZCBodXJ0IHlvdSwNCiAgICBOZXZlciBnb25uYSBnaXZlIHlvdSB1cCwNCiAgICBOZXZlciBnb25uYSBsZXQgeW91IGRvd24sDQogICAgTmV2ZXIgZ29ubmEgcnVuIGFyb3VuZCBhbmQgZGVzZXJ0IHlvdSwNCiAgICBOZXZlciBnb25uYSBtYWtlIHlvdSBjcnksDQogICAgTmV2ZXIgZ29ubmEgc2F5IGdvb2RieWUsDQogICAgTmV2ZXIgZ29ubmEgdGVsbCBhIGxpZSBhbmQgaHVydCB5b3UsDQogICAgLA0KICAgIE5ldmVyIGdvbm5hIGdpdmUsIG5ldmVyIGdvbm5hIGdpdmUsDQogICAgKEdpdmUgeW91IHVwKSwNCiAgICAoT29oKSBOZXZlciBnb25uYSBnaXZlLCBuZXZlciBnb25uYSBnaXZlLA0KICAgIChHaXZlIHlvdSB1cCksDQogICAgLA0KICAgIFdlJ3ZlIGtub3duIGVhY2ggb3RoZXIgZm9yIHNvIGxvbmcsDQogICAgWW91ciBoZWFydCdzIGJlZW4gYWNoaW5nIGJ1dCB5b3UncmUgdG9vIHNoeSB0byBzYXkgaXQsDQogICAgSW5zaWRlIHdlIGJvdGgga25vdyB3aGF0J3MgYmVlbiBnb2luZyBvbiwNCiAgICBXZSBrbm93IHRoZSBnYW1lIGFuZCB3ZSdyZSBnb25uYSBwbGF5IGl0LA0KICAgICwNCiAgICBJIGp1c3Qgd2FubmEgdGVsbCB5b3UgaG93IEknbSBmZWVsaW5nLA0KICAgIEdvdHRhIG1ha2UgeW91IHVuZGVyc3RhbmQsDQogICAgLA0KICAgIE5ldmVyIGdvbm5hIGdpdmUgeW91IHVwLA0KICAgIE5ldmVyIGdvbm5hIGxldCB5b3UgZG93biwNCiAgICBOZXZlciBnb25uYSBydW4gYXJvdW5kIGFuZCBkZXNlcnQgeW91LA0KICAgIE5ldmVyIGdvbm5hIG1ha2UgeW91IGNyeSwNCiAgICBOZXZlciBnb25uYSBzYXkgZ29vZGJ5ZSwNCiAgICBOZXZlciBnb25uYSB0ZWxsIGEgbGllIGFuZCBodXJ0IHlvdSwNCiAgICBOZXZlciBnb25uYSBnaXZlIHlvdSB1cCwNCiAgICBOZXZlciBnb25uYSBsZXQgeW91IGRvd24sDQogICAgTmV2ZXIgZ29ubmEgcnVuIGFyb3VuZCBhbmQgZGVzZXJ0IHlvdSwNCiAgICBOZXZlciBnb25uYSBtYWtlIHlvdSBjcnksDQogICAgTmV2ZXIgZ29ubmEgc2F5IGdvb2RieWUsDQogICAgTmV2ZXIgZ29ubmEgdGVsbCBhIGxpZSBhbmQgaHVydCB5b3UsDQogICAgTmV2ZXIgZ29ubmEgZ2l2ZSB5b3UgdXAsDQogICAgTmV2ZXIgZ29ubmEgbGV0IHlvdSBkb3duLA0KICAgIE5ldmVyIGdvbm5hIHJ1biBhcm91bmQgYW5kIGRlc2VydCB5b3UsDQogICAgTmV2ZXIgZ29ubmEgbWFrZSB5b3UgY3J5LCAg`

This POC is awesome, BTW!! ;-)

**Comment by E3V3A** [comment]
Yeah, we should definitely obfuscate the shellcode. I suggest using HEX encryption or `tac`, so that not even the most 1337 h4x0r can figure out what it does. Although ROT13 would be better, it would be overkill and would cause too much of a performance hit.
